README.md

```diff
@@ -10,4 +10,5 @@ The following meatadata forms are now available for use:
 
 | Parent Software | Form Name |
 |:-:|:-:|
-| TBD | `tbd.xml` |
+| GCAM | `gcam_metadata.xml` |
```

Comment:
Is this for the gcam-hector or stand alone hector? If it is stand alone Hector what sort of role does the GCAM field play?

Reply:
Just stand alone Hector data products.  The GCAM field is in the root README that will inform users of this repository what templates are available for which software.